FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -0,0 +1,291 @@
+// Copyright 2019 Google
```

Comment:
All copyrights should now be Google LLC. See go/copyright

Reply:
Fixed.